docs/api.md

```diff
@@ -2166,6 +2166,7 @@ Shortcut for [page.mainFrame().executionContext().queryObjects(prototypeHandle)]
     - `height` <[number]> height of clipping area
   - `omitBackground` <[boolean]> Hides default white background and allows capturing screenshots with transparency. Defaults to `false`.
   - `encoding` <[string]> The encoding of the image, can be either `base64` or `binary`. Defaults to `binary`.
+  - `captureBeyondViewport` <[boolean]> When true, captures screenshot [beyond the viewport](https://chromedevtools.github.io/devtools-protocol/tot/Page/#method-captureScreenshot). Whe false, falls back to old behaviour, and cuts the screenshot by the viewport size. Defaults to `true`.
```

Comment:
Why did you chose to include this change here? 
Was that a previously undocumented feature?

Reply:
resolved in offline discussion

Reply:
The flag `captureBeyondViewport ` was already announced in [v9.0.0 release notes](https://pptr.dev/#?product=Puppeteer&version=v9.0.0&show=api-release-notes), but wasn't documented. Added documentation in this PR as somewhat screenshot related minor change.